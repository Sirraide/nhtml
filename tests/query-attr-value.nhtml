p[foo=bar]
p[foo=baz]

eval {
    $('[foo=baz]').text = 'baz';
}