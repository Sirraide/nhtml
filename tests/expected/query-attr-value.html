<p foo='bar'></p>
<p foo='baz'>baz</p>
